controller/book.go

```diff
@@ -22,6 +22,19 @@ func NewBookController(db *gorm.DB) *BookController {
 		Usecase: usecase.NewBookUsecase(repository.NewBookRepo(db)),
 	}
 }
+
+// GetBooks godoc
+// @Summary      Get books
+// @Description  Get books
+// @Tags         book
+// @Accept       json
+// @Produce      json
+// @Success      200  {object}  []domain.Book
```

Comment:
a nhớ chỗ này có cái config để hiển thị object mà mình muốn
Hiện tại ntn ở trên doc nó cũng sẽ hiển thị là domain.book thì phải
để đổi lại tên hiển thị thì vào đây https://github.com/swaggo/swag#rename-model-to-display

Reply:
Em có thể đọc thêm phần order các trường để hiển thị theo ý mình muốn. còn mặc định nó sẽ xếp theo alphabet